code-challenges.js

```diff
@@ -20,42 +20,107 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+//  function determines if number below, at, or above boiling point.
+//  paramater: determine where the 3 numbers belong in. 
+//  conditional: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+ 
+
+//const temp1 = 42
+//const temp2 = 350
+//const temp3 = 212
+
+// const nums = (42) => 
+//   for(let i = 0; i < nums; i ++) {
+//     if(nums[i] % 2 == 0)
+//     console.log(`${nums[i]} below boiling point`)  } 
+ 
+//  const nums2 = [212]
+//     for(let i = 0; i < nums2;  i ++) {
+//       if(nums [i] % 2 ==0)
+//       console.log(`${nums2[i]} at boiling point`)}
+
+//   const num3 = [350]
+// for(let i = 0; i < num3; i ++) {
+//   if (nums [i] % 2 ==0)
+//   console.log(`${num3[i]} above boiling point`)
+//}
+
+//second try for this problem
```

Comment:
Nice!  Great refactor for this problem!  Conditional is definitely the best approach here.  Would recommend in the future applying this with the rest of the conditions by using else if and else statements.